google-cloud-storage/src/test/java/com/google/cloud/storage/jqwik/StorageArbitraries.java

```diff
@@ -140,14 +139,21 @@ public static final class Buckets {
 
     private Buckets() {}
 
+    public Arbitrary<Bucket.Lifecycle.Rule.Action> actions() {
+      return Combinators.combine(Arbitraries.of("Delete", "SetStorageClass"), storageClass())
+          .as(
+              (a, s) -> {
+                Bucket.Lifecycle.Rule.Action.Builder actionBuilder =
+                    Bucket.Lifecycle.Rule.Action.newBuilder();
+                actionBuilder.setType(a);
+                if (a.equals("SetStorageClass")) {
+                  actionBuilder.setStorageClass(s);
+                }
+                return actionBuilder.build();
+              });
```

Comment:
I found the previous version easier to read, but if you feel this is easier feel free to keep it.